Containers/nextcloud/entrypoint.sh

```diff
@@ -815,22 +815,5 @@ else
     fi
 fi
 
-# Docker socket proxy
-if version_greater "$installed_version" "27.1.2.0"; then
-    if [ "$DOCKER_SOCKET_PROXY_ENABLED" = 'yes' ]; then
-        if ! [ -d "/var/www/html/custom_apps/app_api" ]; then
-            php /var/www/html/occ app:install app_api
-        elif [ "$(php /var/www/html/occ config:app:get app_api enabled)" != "yes" ]; then
-            php /var/www/html/occ app:enable app_api
-        elif [ "$SKIP_UPDATE" != 1 ]; then
-            php /var/www/html/occ app:update app_api
-        fi
-    else
-        if [ "$REMOVE_DISABLED_APPS" = yes ] && [ -d "/var/www/html/custom_apps/app_api" ]; then
-            php /var/www/html/occ app:remove app_api
```

Comment:
I don't think this is a problem, but the update and delete functionality is not offered by nextcloud_exec_commands.

Reply:
With the addition of this option, it would be possible to manage the required applications more precisely.
`containers-schema.json`
```json
"nextcloud_applications": {
  "type": "array",
  "items": {
    "type": "string",
    "pattern": "^\\w+$"
  }
},
```
If you think it might be good, I can add installation and activation with this option.

Reply:
> ... and delete functionality is not offered by nextcloud_exec_commands.

ah good point. that is why I didnt use it for that in the past. Thanks for reminding me :)

> If you think it might be good, I can add installation and activation with this option.

I think this is a big overengineered for now but thanks for the idea :)